src/actions/dashboard.js

```diff
@@ -48,6 +48,20 @@ export const setCalendarEvents = events => {
   };
 };
 
+export const setCalendarTeamEvents = events => {
+  return {
+    type: actionTypes.SET_CALENDAR_TEAM_EVENTS,
+    payload: events,
+  };
+};
+
+export const clearAllCalendarEvents = () => {
+  return {
+    type: actionTypes.CLEAR_ALL_EVENTS,
+    payload: [],
```

Comment:
do we need a payload here? - not a big deal

Reply:
No, there's no payload here. Could have passed an empty array through, but easier to assign it here.